packages/terra-progress-bar/src/ProgressBar.scss

```diff
@@ -1,4 +1,12 @@
 :local {
+  .progress-bar-gradient {
+    &::-webkit-progress-value {
```

Comment:
Can we use a linear-gradient mixin here so that the code can avoid repetition. Refer to [this](https://gist.github.com/nickpiesco/4731166) source for example.